moveit_ros/robot_interaction/src/interactive_marker_helpers.cpp

```diff
@@ -139,30 +139,37 @@ void addErrorMarker(visualization_msgs::InteractiveMarker& im)
   im.controls.push_back(err_control);
 }
 
+// value for normalized quaternion
+inline constexpr double qv()
```

Comment:
better function name. also why isn't this a constant variable instead of a function?